src/docs/api/public.mdx

```diff
@@ -125,20 +157,22 @@ Write a test [here](https://github.com/getsentry/sentry/tree/master/tests/apidoc
 
 **Tips**:
 
+- `make build-api-docs` will simply attempt to build the OpenAPI JSON. The build will fail if there are any warnings.
 - `make test-api-docs` runs all API docs tests. This command also builds the derefed JSON. This runs in CI but can be useful for local development to make sure everything passes.
```

Comment:
Maybe we could combine the bullet points for `make build-api-docs` and `make test-api-docs` to make clear that `test-api-docs` invokes `build-api-docs` as a prerequisite. (That's what "This command also builds the derefed JSON" is telling us, but the detail seems easy to misunderstand.)